x-pack/legacy/plugins/siem/public/components/page/hosts/host_overview/index.tsx

```diff
@@ -64,39 +68,40 @@ export const HostOverview = React.memo<HostSummaryProps>(
       />
     );
 
-    const descriptionLists: Readonly<DescriptionList[][]> = [
-      [
-        {
-          title: i18n.HOST_ID,
-          description: data.host
-            ? hostIdRenderer({ host: data.host, noLink: true })
-            : getEmptyTagValue(),
-        },
-        {
-          title: i18n.FIRST_SEEN,
-          description:
-            data.host != null && data.host.name && data.host.name.length ? (
-              <FirstLastSeenHost
-                hostname={data.host.name[0]}
-                type={FirstLastSeenHostType.FIRST_SEEN}
-              />
-            ) : (
-              getEmptyTagValue()
-            ),
-        },
-        {
-          title: i18n.LAST_SEEN,
-          description:
-            data.host != null && data.host.name && data.host.name.length ? (
-              <FirstLastSeenHost
-                hostname={data.host.name[0]}
-                type={FirstLastSeenHostType.LAST_SEEN}
-              />
-            ) : (
-              getEmptyTagValue()
-            ),
-        },
-        {
+    const column: DescriptionList[] = [
+      {
+        title: i18n.HOST_ID,
+        description: data.host
+          ? hostIdRenderer({ host: data.host, noLink: true })
+          : getEmptyTagValue(),
+      },
+      {
+        title: i18n.FIRST_SEEN,
+        description:
+          data.host != null && data.host.name && data.host.name.length ? (
+            <FirstLastSeenHost
+              hostname={data.host.name[0]}
+              type={FirstLastSeenHostType.FIRST_SEEN}
+            />
+          ) : (
+            getEmptyTagValue()
+          ),
+      },
+      {
+        title: i18n.LAST_SEEN,
+        description:
+          data.host != null && data.host.name && data.host.name.length ? (
+            <FirstLastSeenHost
+              hostname={data.host.name[0]}
+              type={FirstLastSeenHostType.LAST_SEEN}
+            />
+          ) : (
+            getEmptyTagValue()
+          ),
+      },
+    ];
+    const firstColumn = userPermissions
+      ? column.concat({
```

Comment:
nit - es6
```
[ ...column, {
  title: i18n.MAX_ANOMALY_SCORE_BY_JOB,
  description: (
     <AnomalyScores
@@ -107,8 +112,11 @@ export const HostOverview = React.memo<HostSummaryProps>(
              narrowDateRange={narrowDateRange}
    />),
},
]

Reply:
Fixed it here in these two areas and in a few additional areas from my previous pull request to use the array spread.